x-pack/test/api_integration/apis/metrics_ui/log_entries.ts

```diff
@@ -126,6 +126,55 @@ export default function ({ getService }: FtrProviderContext) {
           expect(messageColumn.message.length).to.be.greaterThan(0);
         });
 
+        it('Returns custom column configurations', async () => {
+          const customColumns = [
+            { timestampColumn: { id: uuidv4() } },
+            { fieldColumn: { id: uuidv4(), field: 'host.name' } },
+            { fieldColumn: { id: uuidv4(), field: 'event.dataset' } },
+            { messageColumn: { id: uuidv4() } },
+          ];
+
+          const { body } = await supertest
+            .post(LOG_ENTRIES_PATH)
+            .set(COMMON_HEADERS)
+            .send(
+              logEntriesRequestRT.encode({
+                sourceId: 'default',
+                startTimestamp: EARLIEST_KEY_WITH_DATA.time,
+                endTimestamp: LATEST_KEY_WITH_DATA.time,
+                center: KEY_WITHIN_DATA_RANGE,
+                columns: customColumns,
+              })
+            )
+            .expect(200);
+
+          const logEntriesResponse = pipe(
+            logEntriesResponseRT.decode(body),
+            fold(throwErrors(createPlainError), identity)
+          );
```

Comment:
could also be written as

```suggestion
          const logEntriesResponse = decodeOrThrow(logEntriesResponseRT)(body);
```